scripts/validate.sh

```diff
@@ -34,3 +34,14 @@ validate_file_nonexists '/var/log/cloud-init-output.log'
 validate_file_nonexists '/var/log/cloud-init.log'
 validate_file_nonexists '/var/log/secure'
 validate_file_nonexists '/var/log/wtmp'
+
+actual_kernel=$(uname -r)
+echo "Verifying that kernel version $actual_kernel matches $KERNEL_VERSION"
```

Comment:
Actual kernel version would look like this: `5.4.141-67.229.amzn2.x86_64`. Do we expect KERNEL_VERSION to match the entire string?

Reply:
No. As you can see by the check below, it adds `*` to the end of the version. We don't care about anything other than verifying what's passed in is present. You will only pass in major.minor version or it will fail at the upgrade step anyway.

Reply:
I missed regex (*) at the end.